banana_dev.egg-info/PKG-INFO

```diff
@@ -1,6 +1,6 @@
 Metadata-Version: 2.1
 Name: banana-dev
-Version: 0.1.0
+Version: 1.0.0
```

Comment:
in ur PR title you said v2.0.0 so is the 1.0.0 intentional here?

Reply:
ohhh good catch